src/ctia/entity/incident.clj

```diff
@@ -126,6 +126,12 @@
 (def incident-mapping
   {"incident"
    {:dynamic false
+    :date_detection false
+    :numeric_detection true
+    :dynamic_templates
+    [{:num
+      {:match_mapping_type "long"
```

Comment:
Scores could be float.
```suggestion
      {:match_mapping_type "float"
```

Reply:
I thought the point of this was to coerce long=>float? IIUC your change is a float=>float coercion?

Reply:
yes you are totally right, ignore my comment :-)